docker-compose.yml

```diff
@@ -4,14 +4,18 @@ services:
     image: postgres
```

Comment:
do we have our own docker file for this or are we using a docker file from somewhere else?

Reply:
we have our own